### ISSUE TITLE
3.0.0-beta002 release

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#933]
Can be labelled **ready** when all other issues destined for this release are closed.

- [x] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
- [x] change `VERSION_SUFFIX` in [AppVeyor](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy/settings) to "-beta002" and initiate a build
- [x] check build
- edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
    - [x] ensure completeness of release notes, including non-owner contributors, if any
    - [x] attach main nupkg and analyzer nupkg
    - [x] publish the release
- [x] push nupkgs to NuGet
- [x] tweet, mentioning contributors
- [x] post link to tweet as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [x] if another pre-release is planned, run `rake pre_release[new_version]` to
    - create a new draft GitHub Release
    - create a new issue (like this one) for the next release, adding it to the next RTM milestone
- [x] close this issue

**Comment by adamralph** [comment]
Now that #904 is done, I guess we're ready to release this beta.

**Comment by thomaslevesque** [comment]
https://twitter.com/fakeiteasyfx/status/805853943972720640
